python/pyiceberg/io/pyarrow.py

```diff
@@ -313,6 +319,19 @@ def _get_fs(self, scheme: str) -> FileSystem:
                 "kerb_ticket": self.properties.get(HDFS_KERB_TICKET),
             }
             return HadoopFileSystem(**client_kwargs)
+        elif scheme in {"gs", "gcs"}:
+            gcs_kwargs: Dict[str, Any] = {}
+            if access_token := self.properties.get(GCS_TOKEN):
+                gcs_kwargs["access_token"] = access_token
+            if expiration := self.properties.get(GCS_TOKEN_EXPIRES_AT):
```

Comment:
If this is interpreted as ms, should the property include `-ms` at the end?

Reply:
Good call